metadata-protobuf/proto/Metaprotocol.proto

```diff
@@ -0,0 +1,155 @@
+syntax = "proto2";
+
+// reacting, unreacting, and changing reaction to video
+message ReactVideo {
+    // ID of the video to react
+    required uint64 video_id = 1;
+
+    // The enum must be wrapped inside "message", otherwide it breaks protobufjs
+    // Reacting again with the same message option will cancel the previous reaction
+    enum Reaction {
+        LIKE = 0;
+        UNLIKE = 1;
+    }
+
+    // Selected reaction
+    required Reaction reaction = 2;
+}
+
+// reacting, unreacting to a comment
+message ReactComment {
+    // ID of the comment to react
+    required string comment_id = 1;
+
+    // ID of the selected reaction
+    required uint32 reaction_id = 2;
+}
+
+// create comment
+message CreateComment {
+    // ID of the video
+    required uint64 video_id = 1;
+
+    // ID of comment member wants to reply (empty if new comment is parent comment)
+    optional string parent_comment_id = 2;
+
+    // Comment text
+    required string body = 3;
+}
+
+// edit comment by author
+message EditComment {
+    // ID of the comment whose text is being edited
+    required string comment_id = 3;
+
+    // New comment body
+    required string new_body = 4;
+}
```

Comment:
Since the fields in the serialized messages are identified by their field numbers, so as long as the field numbers are not conflicting the protobuf de/serialization will keep working fine. However, I have changed the field numbers to start from `1` to remove any sort of confusion.